<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="cmb_white">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="9">
      <widget class="QLabel" name="label_23">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>e</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>c</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="9">
      <widget class="QLabel" name="label_22">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>d</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>f</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>g</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>a</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QIconButton" name="pb_g6">
       <property name="text">
        <string>g6</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QIconButton" name="pb_a5">
       <property name="text">
        <string>a5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QIconButton" name="pb_b5">
       <property name="text">
        <string>b5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QIconButton" name="pb_h6">
       <property name="text">
        <string>h6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QIconButton" name="pb_g7">
       <property name="text">
        <string>g7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QIconButton" name="pb_c5">
       <property name="text">
        <string>c5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QIconButton" name="pb_d6">
       <property name="text">
        <string>d6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QIconButton" name="pb_f7">
       <property name="text">
        <string>f7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QIconButton" name="pb_e7">
       <property name="text">
        <string>e7</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QIconButton" name="pb_b4">
       <property name="text">
        <string>b4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QIconButton" name="pb_f6">
       <property name="text">
        <string>f6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QIconButton" name="pb_a8">
       <property name="text">
        <string>a8</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QIconButton" name="pb_c6">
       <property name="text">
        <string>c6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QIconButton" name="pb_a6">
       <property name="text">
        <string>a6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QIconButton" name="pb_b6">
       <property name="text">
        <string>b6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QIconButton" name="pb_h7">
       <property name="text">
        <string>h7</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QIconButton" name="pb_c4">
       <property name="text">
        <string>c4</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QIconButton" name="pb_e1">
       <property name="text">
        <string>e1</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QIconButton" name="pb_d2">
       <property name="text">
        <string>d2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QIconButton" name="pb_c3">
       <property name="text">
        <string>c3</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QIconButton" name="pb_c2">
       <property name="text">
        <string>c2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QIconButton" name="pb_d5">
       <property name="text">
        <string>d5</string>
       </property>
      </widget>
     </item>
     <item row="8" column="6">
      <widget class="QIconButton" name="pb_f1">
       <property name="text">
        <string>f1</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QIconButton" name="pb_d1">
       <property name="text">
        <string>d1</string>
       </property>
      </widget>
     </item>
     <item row="7" column="6">
      <widget class="QIconButton" name="pb_f2">
       <property name="text">
        <string>f2</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QIconButton" name="pb_e2">
       <property name="text">
        <string>e2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QIconButton" name="pb_d3">
       <property name="text">
        <string>d3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QIconButton" name="pb_b8">
       <property name="text">
        <string>b8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QIconButton" name="pb_d8">
       <property name="text">
        <string>d8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QIconButton" name="pb_c8">
       <property name="text">
        <string>c8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QIconButton" name="pb_f8">
       <property name="text">
        <string>f8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QIconButton" name="pb_h8">
       <property name="text">
        <string>h8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QIconButton" name="pb_d7">
       <property name="text">
        <string>d7</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QIconButton" name="pb_f4">
       <property name="text">
        <string>f4</string>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QIconButton" name="pb_g4">
       <property name="text">
        <string>g4</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QIconButton" name="pb_f3">
       <property name="text">
        <string>f3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QIconButton" name="pb_c7">
       <property name="text">
        <string>c7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QIconButton" name="pb_g5">
       <property name="text">
        <string>g5</string>
       </property>
      </widget>
     </item>
     <item row="7" column="7">
      <widget class="QIconButton" name="pb_g2">
       <property name="text">
        <string>g2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="7">
      <widget class="QIconButton" name="pb_g3">
       <property name="text">
        <string>g3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="8">
      <widget class="QIconButton" name="pb_h4">
       <property name="text">
        <string>h4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QIconButton" name="pb_h5">
       <property name="text">
        <string>h5</string>
       </property>
      </widget>
     </item>
     <item row="6" column="8">
      <widget class="QIconButton" name="pb_h3">
       <property name="text">
        <string>h3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QIconButton" name="pb_e5">
       <property name="text">
        <string>e5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QIconButton" name="pb_d4">
       <property name="text">
        <string>d4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QIconButton" name="pb_f5">
       <property name="text">
        <string>f5</string>
       </property>
      </widget>
     </item>
     <item row="7" column="8">
      <widget class="QIconButton" name="pb_h2">
       <property name="text">
        <string>h2</string>
       </property>
      </widget>
     </item>
     <item row="8" column="8">
      <widget class="QIconButton" name="pb_h1">
       <property name="text">
        <string>h1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QIconButton" name="pb_e4">
       <property name="text">
        <string>e4</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QIconButton" name="pb_b1">
       <property name="text">
        <string>b1</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QIconButton" name="pb_c1">
       <property name="text">
        <string>c1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QIconButton" name="pb_e3">
       <property name="text">
        <string>e3</string>
       </property>
      </widget>
     </item>
     <item row="8" column="7">
      <widget class="QIconButton" name="pb_g1">
       <property name="text">
        <string>g1</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QIconButton" name="pb_b2">
       <property name="text">
        <string>b2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QIconButton" name="pb_a3">
       <property name="text">
        <string>a3</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QIconButton" name="pb_a2">
       <property name="text">
        <string>a2</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QIconButton" name="pb_a1">
       <property name="text">
        <string>a1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QIconButton" name="pb_b3">
       <property name="text">
        <string>b3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QIconButton" name="pb_e6">
       <property name="text">
        <string>e6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QIconButton" name="pb_a7">
       <property name="text">
        <string>a7</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QIconButton" name="pb_a4">
       <property name="text">
        <string>a4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QIconButton" name="pb_b7">
       <property name="text">
        <string>b7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QIconButton" name="pb_e8">
       <property name="text">
        <string>e8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QIconButton" name="pb_g8">
       <property name="text">
        <string>g8</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>b</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>h</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="9">
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="8" column="9">
      <widget class="QLabel" name="label_24">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_25">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>a</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="label_26">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>b</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLabel" name="label_27">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>c</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QLabel" name="label_28">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>d</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QLabel" name="label_29">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>e</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="6">
      <widget class="QLabel" name="label_30">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>f</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="7">
      <widget class="QLabel" name="label_31">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>g</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="8">
      <widget class="QLabel" name="label_32">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>h</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="cmb_black">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QIconButton</class>
   <extends>QPushButton</extends>
   <header>View/ui/widgets/QIconButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
